tests/test_remote_invites.py

```diff
@@ -327,51 +254,46 @@ async def may_invite(self, inviter: str, invitee: str, roomid: str):
             return NOT_SPAM
         return ret[0]  # return first code instead of all of them to make assert easier
 
-    async def test_invite_from_remote_not_on_fed_list(self) -> None:
+    def test_invite_from_remote_not_on_fed_list(self) -> None:
         """Tests that an invite from a remote server not in the federation list gets denied"""
-        self.add_a_contact_to_user_by_token(
```

Comment:
Why did you remove this? Shouldn't you be able to just use the account data event here?